lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -349,19 +351,22 @@ def set_box_aspect(self, aspect, *, zoom=1):
         aspect : 3-tuple of floats or None
             Changes the physical dimensions of the Axes3D, such that the ratio
             of the axis lengths in display units is x:y:z.
+            If None, defaults to (4,4,3).
 
-            If None, defaults to 4:4:3
-
-        zoom : float
-            Control overall size of the Axes3D in the figure.
+        zoom : float, default: 1
+            Control overall size of the Axes3D in the figure. Must be > 0.
         """
+        if zoom <= 0:
+            raise ValueError(f'Argument zoom = {zoom} must be > 0')
+        self._zoom = zoom
```

Comment:
This is not used anywhere, and you made it private now, which means it's also not public API. Either revert to a local variable, or if you would like to be able to read it, add a read-property for that.

I you keep it, add a comment
```suggestion
        # defines self._zoom at object-creation time via
        # AxesBase.__init__ calling set_box_aspect()
        self._zoom = zoom
```

Reply:
> add a read-property for that

I’m not familiar with this in Python and google isn’t helping. What do you mean?

Reply:
Just remove the attribute; it's not used anywhere else.